edge_computing/image_based_upgrade/cnf-understanding-image-based-upgrade.adoc

```diff
@@ -1,7 +1,7 @@
 :_mod-docs-content-type: ASSEMBLY
-[id="understanding-image-based-upgrade-for-sno"]
-= Understanding the image-based upgrade for {sno} clusters
 include::_attributes/common-attributes.adoc[]
+[id="cnf-understanding-image-based-upgrade"]
+= Understanding the image-based upgrade for {sno} clusters
```

Comment:
Also just my opinion but using a definite article here also sounds a bit off, like we're referring to one specific upgrade. Do you think it would sound better without any article and have upgrade be plural?
```suggestion
= Understanding image-based upgrades for {sno} clusters
```

Reply:
I see what you are saying and it does sound better but that is the name of the technology singular "image-based upgrade", so I'd prefer leave it